migrations/versions/d1eae8b9893e_.py

```diff
@@ -0,0 +1,25 @@
+"""add Query.schedule_failures
+
+Revision ID: d1eae8b9893e
+Revises: 65fc9ede4746
+Create Date: 2017-02-03 01:45:02.954923
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'd1eae8b9893e'
+down_revision = '65fc9ede4746'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('queries', sa.Column('schedule_failures', sa.Integer(),
```

Comment:
I wonder if we should store this in the database. Maybe in Redis instead?

The main benefit of keeping it in the database is that we can easily query for it, to generate a report of failing queries (although then maybe we need `last_failure_at` column too?).

The downside is that it makes the table bloated with operational data and adds more calls to the database.